subsys/bluetooth/controller/ll_sw/ull_llcp_local.c

```diff
@@ -140,6 +140,8 @@ static void lr_act_run(struct ll_conn *conn)
 	struct proc_ctx *ctx;
 
 	ctx = lr_peek(conn);
+	LL_ASSERT(ctx != NULL);
```

Comment:
lr_act_run is already guarded at callsite to have a valid context at the head of the queue